src/quo2/components/list_items/token_value/view.cljs

```diff
@@ -0,0 +1,58 @@
+(ns quo2.components.list-items.token-value.view
+  (:require
+    [clojure.string :as string]
+    [quo2.components.icon :as icon]
+    [quo2.components.markdown.text :as text]
+    [quo2.foundations.colors :as colors]
+    [quo2.foundations.resources :as resources]
+    [quo2.theme :as quo.theme]
+    [react-native.core :as rn]
+    [quo2.foundations.common :as common]
+    [quo2.components.list-items.token-value.style :as style]))
+
+(defn- internal-view
+  [{:keys [theme customization-color state status token metrics? values]}]
```

Comment:
let's refer to 'color' as 'customisation-color' as we use it that way for every other component